playbooks/openshift/delete-cluster.yml

```diff
@@ -4,14 +4,28 @@
 #  roles:
 #  - ../../galaxy/openshift-ansible-contrib/roles/openshift-pv-cleanup
 
+- vars_prompt:
+    name: "delete_cluster"
+    prompt: "Are you sure you want to delete the {{env_id}} cluster ] (yes/no)"
+    private: no
+  pre_tasks:
+      - name: "setting the delete_cluster fact"
+        set_fact:
+          delete_cluster: "{{ delete_cluster }}"
+      - name: "setting up the fail module catch"
```

Comment:
Please change to something more descriptive - e.g: `Error out unless user confirms deletion of cluster`